Backend/src/main/java/com/harmonia/backend/service/UserService.java

```diff
@@ -16,6 +16,9 @@
 
 /**
  * Service class for managing users.
+ *
+ * @author Harmonia Team
+ * @version 1.0
  */
 @Service
 public class UserService {
```

Comment:
we need to remember to add documentation here as well or they will be ignored in javadoc